generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management.tsx

```diff
@@ -20,31 +20,82 @@ import * as React from 'react';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import { Button } from 'reactstrap';
+import { Pagination } from 'react-bootstrap';
 import { Translate, ICrudGetAction, TextFormat } from 'react-jhipster';
-import { FaPlus, FaEye, FaPencil, FaTrash } from 'react-icons/lib/fa';
+import { FaPlus, FaEye, FaPencil, FaSort, FaTrash } from 'react-icons/lib/fa';
 
 import { getUsers } from '../../../reducers/user-management';
 import { APP_DATE_FORMAT } from '../../../config/constants';
+import { ITEMS_PER_PAGE } from '../../../shared/util/pagination.constants';
+import { getUrlParameter, IPaginationState } from '../../../shared/util/pagination-utils';
 
 export interface IUserManagementProps {
   getUsers: ICrudGetAction;
   users: any[];
   account: any;
   match: any;
+  totalItems: 0;
 }
 
-export class UserManagement extends React.Component<IUserManagementProps, undefined> {
+export class UserManagement extends React.Component<IUserManagementProps, IPaginationState> {
 
   constructor(props) {
     super(props);
+    const pageParam = getUrlParameter('page', props.location.search);
```

Comment:
you can move this logic to a util method that returns `{ itemsPerPage, sort, order, activePage }` and pass `props.location` to the method as param
so this will become
```
this.state = {
  ...getSortState(props.location)
}
```